tests/python/pants_test/jvm/nailgun_task_test_base.py

```diff
@@ -18,12 +23,4 @@ def setUp(self):
     :API: public
     """
     super(NailgunTaskTestBase, self).setUp()
-    self.set_options(execution_strategy=NailgunTask.ExecutionStrategy.nailgun)
-
-  def tearDown(self):
-    """
-    :API: public
-    """
-    super(NailgunTaskTestBase, self).tearDown()
-    # Kill any nailguns launched in our ephemeral build root.
-    NailgunProcessGroup(metadata_base_dir=self.subprocess_dir).killall()
+    self.set_options(execution_strategy=NailgunTask.ExecutionStrategy.subprocess)
```

Comment:
Would it be ... reasonable to set this on `TaskBase` instead?

Reply:
No. If we can't count on writers to subclass the right test baseclass for the corresponding task or reviewers to catch this all hope is lost.